### ISSUE TITLE
enable hg38 import in production databases

### ISSUE THREAD

**Issue by yichaoS** [cBioPortal/icebox#256]
Manda already did the set up on dashi-dev `cgds_hgnc` database. (migration + expand `reference_genome_genes` table). Need to do the same work for `cgds_gdac` and public database. 

Already have a hg38 (Brain pediatric CHOP) study on datahub here: 
https://github.com/cBioPortal/datahub/tree/master/public/brain_cptac_2020

**Comment by averyniceday** [comment]
@yichaoS is this still something we need to look at?

**Comment by yichaoS** [comment]
@averyniceday Yes, we would need to do it at some point, soonish I assume. We have the prototype imported before on dashi-dev (with the help from @mandawilson for expanding the database) (this is the slack conversation thread https://cbioportal.slack.com/archives/G55LK2SRF/p1633462181007800) and the forntend's "merging different builds warning" feature works well too (see screenshots). But seems like there's more space to improve the visualization still, and a discussion issue regarding to this is still open here: https://github.com/cBioPortal/cbioportal/issues/8919
<img width="716" alt="Screen Shot 2021-10-05 at 3 26 19 PM" src="https://user-images.githubusercontent.com/5973438/154159355-ab12d6e4-1960-4eb3-ad61-2101a8b3780e.png">
<img width="868" alt="Screen Shot 2021-10-05 at 3 27 52 PM" src="https://user-images.githubusercontent.com/5973438/154159362-27809c39-1251-405d-90f9-97e002706c31.png">
